File: app_manager/core/src/main/java/app/documents/core/network/manager/models/request/RequestStorage.java
package app.documents.core.network.manager.models.request;


import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class RequestStorage {

    @SerializedName("url")
    @Expose
    private String url;

    @SerializedName("login")
    @Expose
    private String login;

    @SerializedName("password")
    @Expose
    private String password;

    @SerializedName("token")
    @Expose
    private String token;

    @SerializedName("isCorporate")
    @Expose
    private boolean isCorporate;

    @SerializedName("customerTitle")
    @Expose
    private String customerTitle;

    @SerializedName("providerKey")
    @Expose
    private String providerKey;

    @SerializedName("providerId")
    @Expose
    private String providerId;

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    public String getLogin() {
        return login;
    }

    public void setLogin(String login) {
        this.login = login;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public String getToken() {
        return token;
    }

    public void setToken(String token) {
        this.token = token;
    }

    public boolean getCorporate() {
        return isCorporate;
    }

    public void setCorporate(boolean corporate) {
        isCorporate = corporate;
    }

    public String getCustomerTitle() {
        return customerTitle;
    }

    public void setCustomerTitle(String customerTitle) {
        this.customerTitle = customerTitle;
    }

    public String getProviderKey() {
        return providerKey;
    }

    public void setProviderKey(String providerKey) {
        this.providerKey = providerKey;
    }

    public String getProviderId() {
        return providerId;
    }

    public void setProviderId(String providerId) {
        this.providerId = providerId;
    }
}

File: app_manager/core/src/main/java/app/documents/core/network/manager/models/request/RequestStorage.kt
package app.documents.core.network.manager.models.request

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class RequestStorage(
    @SerializedName("url")
    @Expose
    var url: String? = null,

    @SerializedName("login")
    @Expose
    var login: String? = null,

    @SerializedName("password")
    @Expose
    var password: String? = null,

    @SerializedName("token")
    @Expose
    var token: String? = null,

    @SerializedName("isCorporate")
    @Expose
    var corporate: Boolean = false,

    @SerializedName("customerTitle")
    @Expose
    var customerTitle: String? = null,

    @SerializedName("providerKey")
    @Expose
    var providerKey: String? = null,

    @SerializedName("providerId")
    @Expose
    var providerId: String? = null,

    @SerializedName("isRoomStorage")
    @Expose
    var isRoomStorage: Boolean = false,
)